101-vm-ubuntu-DSVM-GPU-or-CPU/azuredeploy.json

```diff
@@ -0,0 +1,311 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Username for Administrator Account"
+            }
+        },
+        "adminPassword": {
+            "type": "secureString",
+            "metadata": {
+                "description": "Password for for Administrator Account"
+            }
+        },
+        "vmName": {
+            "type": "string",
+            "defaultValue": "[concat('DataScienceVM-', uniqueString(utcNow('F')))]",
```

Comment:
this doesn't need to be globally unique, remove the concat/uniqueString

Reply:
Removed and change to "vmName" by default